repldex/discordbot/commands.py

```diff
@@ -303,8 +303,11 @@ async def random_entry(message):
 
 @betterbot.command(name='ping', aliases=['pong', 'pung'])
 async def ping(message):
+	if message.author.id not in ADMIN_IDS or message.author.id not in DEV_IDS:
```

Comment:
Needs to be an and not an or, because with the or it will only allow ppl with both